projects/vc-api-implementation.md

```diff
@@ -0,0 +1,240 @@
+
+The project was initiated because Energy Web and Elia see a requirement for an API that can be used in the energy sector to issue and verify credentials. Energy Web and Elia believe that an HTTP API lowers the technical barrier to entry for use of SSI, which is important given that organizations which can benefit from SSI often do not have large technical teams dedicated to digital identity. 
+
+Energy Web and Elia chose to implement the VC API specification for the following reasons: 
+
+- An API specification reduces vendor lock-in for consumers choosing to consume the API 
+- The VC API specification is developed in an open process 
+- The VC API specification prioritizes the relevant design goals of modularity, simplicity, composability and extensibility (https://w3c-ccg.github.io/vc-api/#design-goals-and-rationale)
+
+The API has been used by Energy Web and Elia for demonstration projects such as the Rebeam project, in which verifiable credentials were exchanged to authorize electric vehicle charging. Elia’s summary page for Rebeam is available here: https://innovation.eliagroup.eu/en/projects/rebeam. 
+
+Though the project was initiated and is currently being driven by organizations in the energy domain, the implementation is generic and usable across a range of domains.
+
+## Technical information 
+
+The following section describes technical details of the project. *Please note that Elia and Energy Web have some development resources available to make technical changes to the project in order to enable its adoption by the OWF.*
+
+### Technology selections 
+
+| Technology           | Selection  | Selection Rationale |
+|----------------------|------------|----------------|
+| Programming language | Typescript | - Energy Web’s primary development language <br> - Among the most popular programming languages, according to Stack Overflow surveys (https://survey.stackoverflow.co/2023/#most-popular-technologies-language-prof) <br> - Widely used in SSI ecosystem |
+| API framework | Nest.js (https://nestjs.com/) | - Energy Web’s primary web API development framework |
+| Persistence layer | In-memory SQLite | - Ease of setup and development <br> - [A pull request](https://github.com/energywebfoundation/ssi/pull/158) is available which could add support for PostgreSQL |
+| Library: credential signatures | DIDKit (https://www.spruceid.dev/didkit) | - Support for many DID methods <br> - Support for both Data Integrity and JOSE VCs (allowing for future JOSE support for this project) <br> - Packaged as libraries for several programming languages, providing future technical agility |
+
+### Implementation details 
```

Comment:
In this section, there are a number of links to github.com/energywebfoundation. Is the expectation that the referenced tutorials and documentation will move to the OWF?

Reply:
@jrhender  Could the documentation live under the GitHub pages of its own repository?

Reply:
> Could the documentation live under the GitHub pages of its own repository?

@skounis Good idea, I think this could work.